src/redux/saga/fbSignInSaga.ts

```diff
@@ -0,0 +1,56 @@
+import { AccessToken, LoginManager } from 'react-native-fbsdk-next';
+import auth from '@react-native-firebase/auth';
+import { AccessTokenMap } from 'react-native-fbsdk-next/src/FBAccessToken';
+import { createErrorAlertMessageHelper } from '../../helpers/createErrorAlertMessageHelper';
+import { FirebaseDatabaseTypes } from '@react-native-firebase/database';
+import { createReferenceHelper } from '../../helpers/createReferenceHelper';
+import { put } from 'redux-saga/effects';
+import { successSignInAction } from '../actions/authActions/successSignInAction';
+
+export function* fbSignInSaga() {
+  try {
+    // Attempt login with permissions
+    yield LoginManager.logInWithPermissions(['public_profile', 'email']);
+
+    // Once signed in, get the users AccessToken
+    const dataToken: AccessTokenMap = yield AccessToken.getCurrentAccessToken();
+    const accessToken = dataToken.accessToken;
+    // const user = profileRequest.callback();
+    if (!accessToken) {
+      createErrorAlertMessageHelper(
```

Comment:
code below `'if'` should be executed after alert?

Reply:
i replase this to try/catch